tools/ci/update_built.py

```diff
@@ -14,6 +14,12 @@
                              "conformance-checkers/tools/picture.py",
                              "conformance-checkers/tools/url.py"],
     "css-ui": ["css/css-ui/tools/appearance-build-webkit-reftests.py"],
+    # FIXME: These scripts need to be fixed.
```

Comment:
Can you expand what needs to be fixed, or preferably file an issue and replace the FIXME with a link to that issue?